src/main/java/fi/helsinki/cs/tmc/core/commands/RunCheckStyle.java

```diff
@@ -4,6 +4,7 @@
 import fi.helsinki.cs.tmc.langs.abstraction.ValidationResult;
 import fi.helsinki.cs.tmc.langs.domain.NoLanguagePluginFoundException;
 import fi.helsinki.cs.tmc.langs.util.TaskExecutorImpl;
+import java.nio.file.Path;
 
```

Comment:
Import order: this should be below the empty line.

See <https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing>, replacing `com.google` with `fi.helsinki`.